enterprise/backend/src/metabase_enterprise/embed/content_translation/api.clj

```diff
@@ -0,0 +1,37 @@
+(ns metabase-enterprise.embed.content-translation.api
+  "Enterprise endpoints that use [JSON web tokens](https://jwt.io/introduction/) to fetch data needed for embedded
+  content translation. See the documentation for metabase.embedding.api.embed."
+  (:require
+   [metabase.api.macros :as api.macros]
+   [metabase.content-translation.models :as ct]
+   [metabase.embedding.jwt :as embedding.jwt]
+   [metabase.premium-features.core :as premium-features]
+   [metabase.util.i18n :refer [tru deferred-tru]]
+   [metabase.util.log :as log]))
+
+(set! *warn-on-reflection* true)
+
+;;; ------------------------------------------- Content translation --------------------------------------------
+
+(api.macros/defendpoint :get "/dictionary/:token"
+  "Fetch the content translation dictionary via a JSON Web Token signed with the `embedding-secret-key`."
+  [{:keys [token]} :- [:map
+                       [:token string?]]
+   {:keys [locale]}]
+  (log/info "hit embed ct endpoint")
```

Comment:
I don't think this is necessary, pretty sure we already have logging on all requests.